bucket/devspace.json

```
@@ -0,0 +1,46 @@
+{
+    "version": "6.3.2",
+    "description": "DevSpace is an open-source developer tool for Kubernetes that lets you develop and deploy cloud-native software faster",
+    "homepage": "https://devspace.sh",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/devspace-sh/devspace/releases/download/v6.3.2/devspace-windows-amd64.exe",
```

```suggestion
            "url": "https://github.com/devspace-sh/devspace/releases/download/v6.3.2/devspace-windows-amd64.exe#/devspace.exe",
```

You can rename binary inline in the download URL.